src/components/Icon/icons/IconChipDanger/chipDanger.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14 7C14 7.30206 13.7552 7.54688 13.4531 7.54688H11.8489V8.6041H13.4531C13.7552 8.6041 14 8.84901 14 9.15097C14 9.45303 13.7552 9.69785 13.4531 9.69785H11.8489V11.302C11.8489 11.6041 11.6041 11.8489 11.302 11.8489H9.69795V13.4531C9.69795 13.7552 9.45314 14 9.15108 14C8.84901 14 8.6042 13.7552 8.6042 13.4531V11.8489H7.54688V13.4531C7.54688 13.7552 7.30206 14 7 14C6.69794 14 6.45312 13.7552 6.45312 13.4531V11.8489H5.3959V13.4531C5.3959 13.7552 5.15099 14 4.84903 14C4.54697 14 4.30215 13.7552 4.30215 13.4531V11.849H2.69795C2.39589 11.849 2.15108 11.6041 2.15108 11.3022V9.69795H0.546875C0.244812 9.69795 0 9.45314 0 9.15108C0 8.84901 0.244812 8.6042 0.546875 8.6042H2.15108V7.54688H0.546875C0.244812 7.54688 0 7.30206 0 7C0 6.69794 0.244812 6.45312 0.546875 6.45312H2.15108V5.3959H0.546875C0.244812 5.3959 0 5.15099 0 4.84903C0 4.54697 0.244812 4.30215 0.546875 4.30215H2.15108V2.69795C2.15108 2.39589 2.39589 2.15108 2.69795 2.15108H4.30205V0.546875C4.30205 0.244812 4.54686 0 4.84892 0C5.15099 0 5.3958 0.244812 5.3958 0.546875V2.15108H6.45312V0.546875C6.45312 0.244812 6.69794 0 7 0C7.30206 0 7.54688 0.244812 7.54688 0.546875V2.15108H8.6042V0.546875C8.6042 0.244812 8.84901 0 9.15108 0C9.45314 0 9.69795 0.244812 9.69795 0.546875V2.15108H11.302C11.6041 2.15108 11.8489 2.39589 11.8489 2.69795V4.30205H13.4531C13.7552 4.30205 14 4.54686 14 4.84892C14 5.15099 13.7552 5.3958 13.4531 5.3958H11.8489V6.45312H13.4531C13.7552 6.45312 14 6.69794 14 7ZM10.7553 3.24483H3.24483V10.7552H10.7552H10.7553V3.24483ZM8.8082 4.58892C8.97227 4.42368 9.23945 4.42368 9.40469 4.58774C9.56992 4.75181 9.56992 5.01899 9.40469 5.18423L7.59531 6.99946L9.40586 8.81587C9.56992 8.9811 9.56992 9.24829 9.40469 9.41235C9.32266 9.49438 9.21484 9.5354 9.10703 9.5354C8.99922 9.5354 8.89024 9.49438 8.8082 9.41118L7 7.59712L5.1918 9.41001C5.10977 9.49321 5.00078 9.53423 4.89297 9.53423C4.78516 9.53423 4.67734 9.49321 4.59531 9.41118C4.43008 9.24712 4.43008 8.97993 4.59414 8.8147L6.40469 6.99946L4.59414 5.18423C4.43008 5.01899 4.43008 4.75181 4.59531 4.58774C4.76055 4.42368 5.02774 4.42368 5.1918 4.58892L7 6.40181L8.8082 4.58892Z" fill="#F5222D"/>
+</svg>
```

Comment:
If icon have only one color, move `fill` and `fill-opacity` to css because it allows to override them.